services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs

```diff
@@ -214,6 +214,8 @@ impl SentinelS2L2aCogsDataProvider {
                                     ],
                                     RgbaColor::transparent(),
                                     RgbaColor::transparent(),
+                                    None,
+                                    None,
```

Comment:
Please also set under/over instead of default here

Reply:
see above.